vsintegration/src/FSharp.PatternMatcher/SpellChecker.cs

```diff
@@ -13,16 +13,16 @@ internal class SpellChecker
     {
         private const string SerializationFormat = "2";
 
-        public VersionStamp Version { get; }
+        public PatternMatcherVersionStamp Version { get; }
```

Comment:
For my education why is the rename to PatternMatcherVersionStamp an improvement?

Thanks

Kevin

Reply:
Cripes Roslyn's name collides.  That's a bit sucky.

Reply:
Although this is a member on a C# instance.  So I don't really see where the collision would occur?

Reply:
It happened in the IDE tools - perhaps it wouldn't have been an issue at build, but the editor was painted red so I just did this and it fixed it.

Reply:
Which file was showing the red squiggles, I will take a peek?

Reply:
FSharpProjectOptionsManager.fs

Reply:
As explained yesterday this is an issue in the compiler that causes this, as metadata import order matters, which was causing this clash. Hence the rename.